sycl-fusion/jit-compiler/CMakeLists.txt

```diff
@@ -39,13 +38,15 @@ target_link_libraries(sycl-fusion
   ${CMAKE_THREAD_LIBS_INIT}
 )
 
-if(NOT MSVC AND NOT APPLE)
-  # Manage symbol visibility through the linker to make sure no LLVM symbols
-  # are exported and confuse the drivers.
-  set(linker_script "${CMAKE_CURRENT_SOURCE_DIR}/ld-version-script.txt")
-  target_link_libraries(
-    sycl-fusion PRIVATE "-Wl,--version-script=${linker_script}")
-  set_target_properties(sycl-fusion PROPERTIES LINK_DEPENDS ${linker_script})
+if (BUILD_SHARED_LIBS)
+  if(NOT MSVC AND NOT APPLE)
+    # Manage symbol visibility through the linker to make sure no LLVM symbols
+    # are exported and confuse the drivers.
+    set(linker_script "${CMAKE_CURRENT_SOURCE_DIR}/ld-version-script.txt")
+    target_link_libraries(
+      sycl-fusion PRIVATE "-Wl,--version-script=${linker_script}")
+    set_target_properties(sycl-fusion PROPERTIES LINK_DEPENDS ${linker_script})
+  endif()
 endif()
 
 install(TARGETS sycl-fusion
```

Comment:
Should we install the library at all in case we are building a static library? 

This would also affect [the list of components to install](https://github.com/intel/llvm/blob/sycl/sycl/source/CMakeLists.txt#L96), in case we decide not to install the static library.

Reply:
good point, I'll fix that

Reply:
addressed both point